### ISSUE TITLE
Replace links to hru.gov

### ISSUE THREAD

**Issue by humancompanion** [USAJOBS/Help#326]
hru.gov has been decommissioned. Thus we need to update links in the Help Center to hru this week (next release goes out Friday thus we really need to give a build to dev as soon as possible).

Here are the pages:

* [x] working-in-government/unique-hiring-paths/peace-corps/index.html
* [x] working-in-government/fair-and-transparent/financial-suitability/index.html
* [x] working-in-government/fair-and-transparent/signature-false-statements/index.html
* [x] faq/application/documents/resume/what-to-include/index.html

@dalenard21 - Hopefully you can find similar content elsewhere. If not, let's just remove the links.

@dreeeeewwwww - Let's use this tiny fire-drill to show you how to build the Help Center.

**Comment by dalenard21** [comment]
@humancompanion 

I removed all of the links.  I tried searching for these files on opm.gov and they're still linking to the old HRU site.  Michelle is going to email someone the files that we no longer have access to and find out if or when we'll be able to link to them.

**Comment by KatMullens-work** [comment]
Hi @humancompanion : I can try to follow up since Michelle is out. Carmen's team is building a new site with similar content but we didn't have a launch date for it. Is there anything else I can help with?

**Comment by dalenard21** [comment]
@KatMullens-work  and @humancompanion 

Julie and I met with RPO a couple of weeks ago and their new site won't be ready for about a year.

I did email Michelle the pages before she went out on leave, but did not hear back from her.  She had more pressing items on her plate and since the new site won't be live for awhile, I don't think there's anything else to do.

I say we close this issue since we removed the broken links. 

Thanks
